docker/base-builder/Dockerfile

```diff
@@ -21,9 +21,13 @@ FROM gcr.io/fuzzbench/base-image
 # recommended for some reason.
 RUN apt-get update -y && apt-get install -y --no-install-recommends \
     automake \
+    bison \
```

Comment:
I've just stopped installing dependencies in this file unless it is very generally useful.
cmake and unzip might be appropriate here but I'd rather everything gets installed in build.sh.
Sorry to change things on you in the middle.
Here's an example of how I do this: https://github.com/google/fuzzbench/blob/master/benchmarks/freetype2-2017/build.sh#L18
(we don't have sudo in the images so the script assumes running as root, Do you need these to run on your host?)

Reply:
Fixed.